### PR TITLE
Completed express-notes-api exercise.

### PR DESCRIPTION
**Pull request by alexei-dayrit**
https://user-images.githubusercontent.com/87846307/159394558-77023cf5-4db2-48fc-b3a5-5c95a5372d1d.mov

https://user-images.githubusercontent.com/87846307/159394562-c1e7d3d6-60ea-4ab5-8612-a0b12fb13b76.mov

https://user-images.githubusercontent.com/87846307/159394568-290439c1-9008-42af-bf93-36aa00ec411d.mov

https://user-images.githubusercontent.com/87846307/159394578-93e0b98f-6f02-4de7-9a2a-da95b3871ecb.mov

https://user-images.githubusercontent.com/87846307/159394583-3f4d0343-6c8d-460b-8b15-c854c218ab84.mov

### CODE REVIEW

File: express-notes-api/index.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      error: 'Cannot find note with id ' + req.params.id
+    });
+  } else if (parseInt(req.params.id) === parsedData.notes[req.params.id].id) {
+    res.status(200).json(parsedData.notes[req.params.id]);
+  }
+});
+
+app.post('/api/notes', (req, res) => {
+  const nextIdName = parsedData.nextId;
+  const nextIdNum = parsedData.nextId;
```
Comment (thebearingedge) [inline express-notes-api/index.js:34]: I don't really understand why you need both of these variables.

File: express-notes-api/index.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    });
+  } else if (parsedData.notes[req.params.id] === undefined) {
+    res.status(404).json({
+      error: 'Cannot find note with id ' + req.params.id
+    });
+  } else if (parseInt(req.params.id) === parsedData.notes[req.params.id].id) {
+    delete parsedData.notes[req.params.id];
+    const note = JSON.stringify(parsedData, null, 2);
+    fs.writeFile('./derp/data.json', note, err => {
+      if (err) {
```
Comment (thebearingedge) [inline express-notes-api/index.js:77]: Be sure to change this back to just `'./data.json'` in all of your `fs.writeFile` calls.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Great, just some minor cleanup.

**Comment by alexei-dayrit** [comment]
https://user-images.githubusercontent.com/87846307/159547889-0053304a-4930-497b-9811-a46d29d0fcd2.mov

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**